decore.rb

```diff
@@ -0,0 +1,34 @@
+class Nameable
+  def correct_name
+    raise 'NotImplementedError'
+  end
+end
```

Comment:
Well done! You create Namable class with a method 'correct_name' that raises an error 🔥